packages/browser-webdriverio/src/commands/click.ts

```diff
@@ -4,10 +4,10 @@ import type { UserEventCommand } from './utils'
 export const click: UserEventCommand<UserEvent['click']> = async (
   context,
   selector,
-  options = {},
+  options,
```

Comment:
Did you check what is passed here when `= {}` is removed? I think we always normalize it in the `browser/*/locators/index.ts`

Maybe a better test could catch that 😄

Reply:
ah good question, I'll double check at least by adding logs :)

I'm not sure how to do a good test, as they should essentially do the same thing in the end ... except with more work (possibly taking longer especially).

Reply:
or maybe I could try to mock the call to see what is _really_ called...

Reply:
Playwright defaults to `{}` too, and the `preview` one doesn't take any option, so I guess we can remove the default everywhere.

https://github.com/microsoft/playwright/blob/e7bff526433b6dcb02801763ab5b1c6407902d47/packages/playwright-core/src/client/locator.ts#L108

Reply:
Sure, let's not add a new test specifically for this, but it's also important that already established tests don't fail 😄 I can see there is now only `click with modifier` on firefox failing for some reason 🤔

Reply:
yeah I was looking into that too...

https://github.com/vitest-dev/vitest/blob/1f730373858a7d7f49785cca5d6c45d17afc77b6/test/browser/fixtures/user-event/keyboard.test.ts#L65-L67

I've been told from webdriver's firefox implementor that it's expected: this works with actions but not elementClick at the moment. Which is quite unfortunate :-( But they said also that they didn't implement it because the other browsers didn't either.

It's not clear to me if the test runs in Chrome to check this assumption is true? I don't see it in the test output, but also I see nothing in the file that would run it only in Firefox...

Reply:
I think if we can add a documentation notice to the `click` event, it's fine to merge. The impact should not be big

Reply:
oh no, should be fixed now :-)